libs/@hashintel/design-system/src/components.ts

```diff
@@ -16,12 +16,15 @@ export * from "./icon-barcode";
 export * from "./icon-button";
 export * from "./icon-circle-plus";
 export * from "./icon-close";
+export * from "./icon-download";
+export * from "./icon-file";
```

Comment:
Why are we adding these to the design system? I've been sticking new FA HASH app icons into `apps/hash-frontend/src/shared/icons/`, but happy to adapt my behaviour if the design system is a better place for them.

However it would be nice for them to have their own directory to avoid both components and icons living in the `src` directory , and for us to be consistent with how we name them between the HASH app and the design system.

Reply:
Only really in anticipation of us wanting to use them in other libraries, e.g. the type editor, in which case it saves us moving them later (and it makes no real difference otherwise where they are). We can move them to a different directory, yes.